### ISSUE TITLE
Not all docs replicating with p2p .NET pull replication

### ISSUE THREAD

**Issue by sethrosetter** [couchbase/couchbase-lite-net#765]
Using `.NET 1.3.1-13`. 

Failing test - https://github.com/couchbaselabs/mobile-testkit/blob/master/testsuites/listener/shared/client_client/test_peer_2_peer_sanity.py#L229-L269

Only a subset of the documents replicate before the test timesout.

To reproduce, run the following

```
pytest  -s --liteserv-one-platform=net \
        --liteserv-one-version=1.3.1-13 \
        --liteserv-one-host=192.168.0.14 \
        --liteserv-one-port=50000 \
        --liteserv-one-storage-engine=SQLCipher \
        --liteserv-two-platform=android \
        --liteserv-two-version=1.3.1-61 \
        --liteserv-two-host=192.168.0.55 \
        --liteserv-two-port=60000 \
        --liteserv-two-storage-engine=SQLCipher \
        -k "test_peer_2_peer_sanity_pull" \
        testsuites/listener/shared/client_client/
```

**Comment by sethrosetter** [comment]
Failing test - http://uberjenkins.sc.couchbase.com/job/net-listener-client-client/17/testReport/testsuites.listener.shared.client_client/test_peer_2_peer_sanity/test_peer_2_peer_sanity_pull/

**Comment by borrrden** [comment]
Confirmed failing with 1.3.1 and passing with 1.4.0-46 (but only if run manually and not through pytest for some reason....)

**Comment by borrrden** [comment]
There are two downstream issues that need addressing in relation to this:

https://github.com/couchbase/couchbase-lite-java-core/issues/1495
https://github.com/couchbase/couchbase-lite-ios/issues/1494

(Ironically the issue numbers are sequential even though they are different repos, hah!)

**Comment by borrrden** [comment]
If running .NET <-> Android or .NET <-> .NET this should be fixed now

**Comment by borrrden** [comment]
@sethrosetter Looks like the iOS downstream issue has been closed as well.  This test should be passing everywhere now.

**Comment by borrrden** [comment]
Closed in 7dd973406491c76c7db1d247ebb23454b404d2cd
